synapse/util/caches/descriptors.py

```diff
@@ -455,28 +457,23 @@ def complete_all(res: Dict[Hashable, Any]) -> None:
                         deferreds_map[e].callback(val)
                         results[e] = val
 
-                def errback(f: Failure) -> Failure:
-                    # the wrapped function has failed. Invalidate any cache
-                    # entries we're supposed to be populating, and fail
-                    # their deferreds.
-                    for e in missing:
-                        key = arg_to_cache_key(e)
-                        cache.invalidate(key)
```

Comment:
this seems to have been redundant (since the cache will invalidate the entry itself when we `errback` the deferred), so I have removed it